spec/jobba_spec.rb

```diff
@@ -14,4 +14,19 @@
     expect(Jobba.all.count).to eq 6
   end
 
+  it 'can cleanup old statuses' do
+    Jobba.clear_all_jobba_data!
```

Comment:
This happens in an `after_each` for all specs -- that sufficient?

Reply:
Hm probably. I'll try without.